platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/provider/DefaultValueSourceProviderFactory.java

```diff
@@ -231,9 +237,8 @@ public boolean isImmutable() {
             return true;
         }
 
-        @Nullable
-        public Try<T> getObtainedValueOrNull() {
-            return value.value;
```

Comment:
💭 I had to kill this method, I just couldn't make myself to write `return value.value.getValue()`.